sync.go

```diff
@@ -102,6 +102,9 @@ func (s *SyncAction) prepareArtifact(username, password, override string) error
 		s.saveKeyring = true
 	}
 
+	override = truncateOverride(override)
+	log.Info("Truncated override value to 7 chars: %s", override)
```

Comment:
do we really need to print it every time? maybe move it to truncateOverride, under true condition ?

Reply:
good point !